trestle/tasks/csv_helper.py

```diff
@@ -28,22 +28,41 @@ class Column():
 
     def __init__(self) -> None:
         """Initialize."""
-        self.columns = [
+        self._columns_required = [
             'Rule_Id',
             'Rule_Description',
             'Profile_Reference_URL',
             'Profile_Description',
             'Component_Type',
             'Control_Mappings',
             'Resource',
-            'Parameter_Id',
-            'Parameter_Description',
-            'Parameter_Default_Value',
-            'Parameter_Value_Alternatives'
         ]
-        self.help_list = []
-        for column in self.columns:
-            self.help_list.append(column)
+        self._columns_optional = [
+            'Parameter_Id', 'Parameter_Description', 'Parameter_Default_Value', 'Parameter_Value_Alternatives'
+        ]
```

Comment:
There are other optional columns also that we decided earlier like fetchers, checkers, etc. Please see the best practices spreadsheet to CSV file that you generated earlier.

Reply:
Fixed.
optional columns:   Parameter_Id
                      Parameter_Description
                      Parameter_Default_Value
                      Parameter_Value_Alternatives
                      Check_Id
                      Check_Description
                      Fetcher
                      Fetcher_Description